cibuildwheel/macos.py

```diff
@@ -270,7 +270,7 @@ def setup_python(
     config_is_arm64 = python_configuration.identifier.endswith("arm64")
     config_is_universal2 = python_configuration.identifier.endswith("universal2")
 
-    if python_configuration.version == "3.9":
+    if python_configuration.version not in {"3.6", "3.7", "3.8"}:
```

Comment:
I've updated this but I can also think of the following even though it doesn't read as easily:
```suggestion
    if tuple(map(int, python_configuration.version.split("."))) >= (3, 9):
```

Reply:
I prefer the `not in`, personally, I think it reads better.